scripts/test.py

```diff
@@ -213,6 +213,32 @@ def test(name, to_run, want, fail=False, sort=False):
 
 # END OF REGRESSION TESTS
 
+# Cache test
+to_run = """
+curl https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-04.csv -o taxi.csv;
+./dsq --cache taxi.csv "SELECT passenger_count, COUNT(*), AVG(total_amount) FROM {} GROUP BY passenger_count";
```

Comment:
Although the original script doesn't include it you should add an `ORDER BY  COUNT(*) DESC` otherwise by the SQL spec the order could be random. I don't know if SQLite picks an order automatically or if it's really allowed to be random.